packages/react/src/combobox/ComboboxBase/ComboboxTag.tsx

```diff
@@ -30,7 +30,7 @@ export const ComboboxTag = forwardRef<HTMLSpanElement, ComboboxTagProps>(
 				gap={0.25}
 				fontSize="sm"
 				color="text"
-				focusRingFor="keyboard"
+				focusRingFor="all"
```

Comment:
Previously no way to know that you had pressed a tag as we have the functionality of being able to press tag then press delete key to remove a tag.

Reply:
Interesting. Should this be a button (Afghanistan. Remove) then? That would resolve my other issue where you can navigate to it, but it's just a group. Or is that not how they work? Where even is the spec for this kind of combobox?

Reply:
>Should this be a button (Afghanistan. Remove) then?

A button doesn't feel right to me because you can't press it as normal and the screen reader won't know to announce that. I've extended the group's name to include instruction that you can press the delete key to remove the tag. Let me know what you think.

>Where even is the spec for this kind of combobox?

This was shared with us in the audit but it's different to most multi select comboboxes I've seen. https://codepen.io/smhigley/pen/gObMVzv

Further, the most common multi select combobox must be the "To" field of email clients. Our pattern is more closely aligned with that.